cdap-ui/app/cdap/components/Administration/index.js

```diff
@@ -45,7 +45,8 @@ class Administration extends Component {
     platforms: [],
     uptime: 0,
     loading: true,
-    currentTab: objectQuery(this.props.location, 'state', 'showConfigTab') ? ADMIN_TABS.config : ADMIN_TABS.management
+    currentTab: this.props.location.pathname.indexOf('/configuration') !== -1 ? ADMIN_TABS.config : ADMIN_TABS.management,
+    accordionToExpand: typeof this.props.location.state === 'object' ? this.props.location.state.accordionToExpand : null
```

Comment:
instead of doing this, you should switch to rendering the tab content with `Routes`